galleon-pack/src/main/resources/layers/standalone/cloud-server/layer-spec.xml

```diff
@@ -1,13 +1,10 @@
 <?xml version="1.0" ?>
 <layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="cloud-server">
     <dependencies>
-        <layer name="cloud-profile"/>
-        <layer name="core-server"/>
-        <layer name="core-tools"/>
+        <layer name="jaxrs-server"/>
+        <layer name="ee-security" optional="true"/>
+        <layer name="jms-activemq" optional="true"/>
+        <layer name="observability" optional="true"/>
+        <layer name="resource-adapters" optional="true"/>
     </dependencies>
-    <feature-group name="undertow-elytron-security"/>
```

Comment:
@jfdenise could you please confirm my understanding?
 
* `core-server`, `core-tools` and `undertow-elytron-security` will now come from `jaxrs-server` > `datasources-web-server`
*  `cloud-profile` is no longer required as all layers listed in it are now listed in `cloud-server`. But shouldn't still be a relation between `cloud-server` and `cloud-profile`?

Reply:
What kind of relation we could have? cloud-profile dependency is to me an implementation detail. With dependency on jaxrs-server we offer the same level of flexibility.

Reply:
This is a tangent and not something to address here but I suspect the documentation of cloud-profile is no longer accurate.

cloud-profile, an aggregation of the basic layers (except h2-* layers) and a web-server .

The "basic layers" list is a moving target.

Hmm, maybe my #12927 change to that file (https://github.com/wildfly/wildfly/pull/12927/files#diff-9a6f6b7a6bd30fddb786db3840aac00f) somewhat makes that go away by moving FT/JWT/OpenAPI into a different section.  But that text can break in the future, as soon as we add a new layer.

Perhaps drop it from the doc page?

Reply:
@jfdenise ^^^

Reply:
Yes, +1 to remove it from the doc.